### PR TITLE
aliyun-ack-en translated

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: adakisme <serenashe@yunify.com>

### CODE REVIEW

File: content/zh/docs/multicluster-management/import-cloud-hosted-k8s/import-aliyun-ack.md [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
 
    ```yaml
    jwtSecret: "QVguGh7qnURywHn2od9IiOX6X8f8wK8g"
    ```
 
-2. Log in to the KubeSphere console of the ACK cluster as `admin`. Click **Platform** in the upper left corner and then select **Clusters Management**.
+2. 以 `admin` 身份登录 ACK 集群的 KubeSphere 控制台。点击左上角的**平台管理**，选择**集群管理**。
 
-3. Go to **CRDs**, input `ClusterConfiguration` in the search bar, and then press **Enter** on your keyboard. Click **ClusterConfiguration** to go to its detail page.
+3. 访问**自定义资源 CRD**，在搜索栏输入  `ClusterConfiguration`，然后按下键盘上的**回车键**。点击 **ClusterConfiguration** 访问其详情页。
```
Comment (Felixnoo) [inline content/zh/docs/multicluster-management/import-cloud-hosted-k8s/import-aliyun-ack.md:34]: Delete the extra space between 输入 and `ClusterConfiguration`

**Review by Felixnoo — CHANGES_REQUESTED** [review]

File: content/zh/docs/multicluster-management/import-cloud-hosted-k8s/import-aliyun-ack.md [hunk 2]
```diff
@@ -1,45 +1,45 @@
 ---
-title: "Import an Aliyun ACK Cluster"
+title: "导入阿里云 ACK 集群"
 keywords: 'Kubernetes, KubeSphere, multicluster, Aliyun ACK'
 description: 'How to import an Aliyun ACK Cluster'
 titleLink: "Import an Aliyun ACK Cluster"
```
Comment (Sherlock113) [inline content/zh/docs/multicluster-management/import-cloud-hosted-k8s/import-aliyun-ack.md:5]: Translate the value of this three lines

**Comment by Sherlock113** [comment]
Merge these five commits please. Use `git log` to check your commit history.

**Comment by Sherlock113** [comment]
Please submit this pr to the release-3.0 branch as well.

**Pull request merged**